test/integration/element.test.js

```diff
@@ -0,0 +1,297 @@
+import {getBrowser} from '../mini-testium-mocha';
+import assert from 'assertive';
+
+function stripColors(message) {
+  return message.replace(/\u001b\[[^m]*m/g, '');
+}
+
+xdescribe('element', () => {
```

Comment:
Yep, all but `navigation` and `cookie` are pending (because only the navigation and cookie APIs are ported yet).